build.gradle

```diff
@@ -145,22 +156,19 @@ jooq {
             generationTool {
                 logging = org.jooq.meta.jaxb.Logging.WARN
                 jdbc {
-                    driver = 'org.mariadb.jdbc.Driver'
-                    //ُTODO: Make this configurable
-                    url = System.getenv("JDBC_URL")
-                    user = System.getenv("DATA_BASE_USER_NAME")
-                    password = System.getenv("DATA_BASE_PASSWORD")
+                    driver = 'org.postgresql.Driver'
+                    url = "jdbc:postgresql://localhost:5432/postgres"
```

Comment:
Maybe we can store it in env var ?

Reply:
I had issue with that in gradle. It causes an error with the work flow

Reply:
Fixed